cobalt/docker/linux/Dockerfile

```diff
@@ -31,13 +31,9 @@ RUN apt-get update && apt-get install -y \
     jq \
     && rm -rf /var/lib/apt/lists/*
 
-# === Install portable sccache binary
-ARG SCCACHE=sccache-v0.3.0-x86_64-unknown-linux-musl.tar.gz
-ARG SCCACHE_SHA256=e6cd8485f93d683a49c83796b9986f090901765aa4feb40d191b03ea770311d8
+ARG GCLOUD=google-cloud-cli-linux-x86_64.tar.gz
```

Comment:
Does this need to be an arg? Seems to me this is always going to be the filename?

Reply:
Never mind. It's to not have to repeat the filename below.